src/plugins/vis_type_timeseries/server/types.ts

```diff
@@ -6,8 +6,24 @@
  * Side Public License, v 1.
  */
 
-import type { IRouter } from 'src/core/server';
-import type { DataRequestHandlerContext } from '../../data/server';
+import type { IRouter, FakeRequest, IUiSettingsClient } from 'src/core/server';
+import type {
+  DataRequestHandlerContext,
+  EsQueryConfig,
+  IndexPatternsService,
+} from '../../data/server';
+import { VisPayload } from '../common/types';
+import { SearchStrategyRegistry } from './lib/search_strategies';
 
 export type VisTypeTimeseriesRequestHandlerContext = DataRequestHandlerContext;
 export type VisTypeTimeseriesRouter = IRouter<VisTypeTimeseriesRequestHandlerContext>;
+export type VisTypeTimeseriesRequest<TPayload = unknown> = FakeRequest & { body: TPayload };
```

Comment:
I think instead of FakeRequest here should be KibanaRequest

Reply:
especially if below you anyway cast it to KibanaRequest
![image](https://user-images.githubusercontent.com/20072247/109791974-1355f780-7c24-11eb-9fa8-405904df075d.png)

Reply:
updated